### PR TITLE
ECU page updates

### PR DESCRIPTION
**Pull request by jhdalek55**
Added images and did some capitalization changes.

### CODE REVIEW

File: ecus.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ css_id: ecus
 
 At the highest level, the basic requirement for an ECU to be capable of supporting Uptane is that it must be able to perform either full or partial verification, and have a secure source of time. (See the [Uptane Standard](https://uptane.github.io/uptane-standard/uptane-standard.html#build-time-prerequisite-requirements-for-ecus) for official requirements.)
 
-To bootstrap an Uptane-capable ECU, a few things need to be provisioned into the ECU:
+To bootstrap an Uptane-capable ECU, a few things need to be provisioned into unit:
```
Comment (patrickvacek) [inline ecus.md]: "it"?

Comment (jhdalek55) [reply ecus.md]: yes. Thanks for the catch.

Comment (jhdalek55) [reply ecus.md]: I'll check on the graphic. It was loaded to the repo, but perhaps I did not properly enter the image name.

**Comment by jhdalek55** [comment]
The main update here was just adding a figure. @patrickvacek  has already checked. Can we get one more review and then merge?

**Pull request merged**